waspc/src/Wasp.hs

```diff
@@ -133,6 +134,15 @@ setApp wasp app = wasp { waspElements = (WaspElementApp app) : (filter (not . is
 fromApp :: App -> Wasp
 fromApp app = fromWaspElems [WaspElementApp app]
 
+-- * Auth
+
+getAuth :: Wasp -> Maybe Wasp.Auth.Auth
+getAuth wasp = let auths = [a | WaspElementAuth a <- waspElements wasp] in 
+    case auths of
+        []  -> Nothing
+        [a] -> Just a
+        _   -> error "Wasp has to contain exactly one WaspElementAuth element!"
```

Comment:
Actually it can also contain 0 :D! So maybe we can say: it can't contain more than one ...